### PR TITLE
[HPT-871] Copies JP2 to derivatives folder on file_set creation.

### PR DESCRIPTION
**Pull request by andjsmit**
Also adds ruby version files to git ignore files.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -22,3 +22,8 @@ noid-minter-state
 /fits
 /coverage
 /staged_files
+
+# Ignore Ruby Version Management stuff
+# /.rvmrc
+/.ruby-*
```
Comment (mwoodiupui) [inline .gitignore:28]: Yes, please!

**Comment by wang58** [comment]
Looks good to me.

**Comment by randalldfloyd** [comment]
@andjsmit Just curious, what happens to the filename that gets put into both the title and the label on the fileset? It seems that doing this would name the fileset after the derivative JP2K file it was given for ingest. Does that also mean the JP2K gets stored as the content node? It got me thinking about some things, like maybe the label/title should knock off the file extension, and then the fileset could have two named redirect nodes, one to resolve to the image server and one to resolve to the master location.

**Comment by andjsmit** [comment]
The title does end up being the original file name extension and all. I see this in both the GUI and the fedora title field. Not sure about the label. It appears empty in the file edit form. NO WAIT. In the gui it is the same value but the label style has it appearing as white, same as background. Found a bug.

As for the content node: do you mean does the JP2 file get stored in Fedora? If so, yes that is the case.

**Review by randalldfloyd — APPROVED** [review]
This seems like valid and desirable behavior even if we don't use it here at IU.

**Pull request merged**